### PR TITLE
Added f90nml (Fortran 90 namelist parser) recipe files

### PR DESCRIPTION
**Pull request by cpelley**

### CODE REVIEW

File: recipes/f90nml/bld.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install 
+if errorlevel 1 exit 1
```
Comment (ocefpaf) [inline recipes/f90nml/bld.bat]: Please drop both `bld.bat` and `build.sh` in favor of the `script` line. (See the example recipe.)

File: recipes/f90nml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package:
+  name: f90nml
+  version: "0.21"
```
Comment (ocefpaf) [inline recipes/f90nml/meta.yaml]: Please use a jinja variable for the version number to make it easier to maintain the recipe.

File: recipes/f90nml/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+source:
+  fn: f90nml-0.21.tar.gz
+  url: https://pypi.python.org/packages/60/9d/2b784b60519009fb2b2563190e22dc8e169c7238c63caa981cf1fed15a47/f90nml-0.21.tar.gz
+  md5: 12877413c95b9d34301b1787344b65b2
```
Comment (ocefpaf) [inline recipes/f90nml/meta.yaml]: Please change these lines to:

```yaml
url: https://pypi.io/packages/source/f/f90nml/f90nml-{{ version }}.tar.gz
sha256: df0538bfd977f6e72d3bbf6beb8bd7dc7c01882d5a89e99fbfdfbf1aa5b24b8b

```

File: recipes/f90nml/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+about:
+  home: http://github.com/marshallward/f90nml
+  license: Apache v2.0
+  summary: 'Fortran 90 namelist parser'
+  license_family: APACHE
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
```
Comment (ocefpaf) [inline recipes/f90nml/meta.yaml]: Please remove the boilerplate comments.

**Comment by cpelley** [comment]
Thank for looking at this @ocefpaf, appreciated.
I have made some changes to reflect the above.

Cheers

File: recipes/f90nml/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+{% set version = "0.21" %}
+{% set sha256 = "df0538bfd977f6e72d3bbf6beb8bd7dc7c01882d5a89e99fbfdfbf1aa5b24b8b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/60/9d/2b784b60519009fb2b2563190e22dc8e169c7238c63caa981cf1fed15a47/{{ name }}-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/f90nml/meta.yaml]: One last thing, can you change the URL to a more readable

`https://pypi.io/packages/source/f/f90nml/f90nml-{{ version }}.tar.gz`

It is up to you to use `{{ name[0] }}/{{ name }}/{{ name[0] }}-` too. I like only the version as a jinjs variable to make it easier to copy-n-past the URL.

**Comment by cpelley** [comment]
Thanks @ocefpaf, done.

File: recipes/f90nml/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  # Python imports
```
Comment (ocefpaf) [inline recipes/f90nml/meta.yaml]: You missed this comment but we can remove that later. Let's get this merged!

**Comment by cpelley** [comment]
Thanks @ocefpaf :)

**Pull request merged**